### PR TITLE
fix depedencies which are broken for Arm64 builds

### PR DESCRIPTION
**Pull request by AWSjswinney**
- update BUF version
- disable Google Cloud SDK for Arm

### CODE REVIEW

File: docker/build-tools/Dockerfile [hunk 1]
```diff
@@ -258,7 +258,13 @@ RUN chmod 555 ${OUTDIR}/usr/bin/kpt
 # Install gcloud command line tool
 # Install gcloud beta component
 RUN set -eux; \
-    GCLOUD_TAR_FILE="google-cloud-sdk-${GCLOUD_VERSION}-linux-$(uname -m).tar.gz"; \
+    \
+    case $(uname -m) in \
+        x86_64)  export GCLOUD_TAR_FILE="google-cloud-sdk-${GCLOUD_VERSION}-linux-$(uname -m).tar.gz" ;; \
+        aarch64) echo "arm not yet supported"; exit 0 ;; \
```
Comment (howardjohn) [inline docker/build-tools/Dockerfile:264]: Whats wrong with ARM? It doesn't build or it doesn't run after build? It does seem to be supported: https://cloud.google.com/sdk/docs/install#linux

**Review by howardjohn — APPROVED** [review]

File: docker/build-tools/Dockerfile (hunk 1 above)
Comment (AWSjswinney) [reply docker/build-tools/Dockerfile:264]: I have been getting a 404 when I run the wget. I haven't checked today, though.

Comment (AWSjswinney) [reply docker/build-tools/Dockerfile:264]: This is the error message when building from master with the BUF version updated:
```
 => ERROR [binary_tools_context 90/95] RUN set -eux;     GCLOUD_TAR_FILE="google-cloud-sdk-345.0.0-linux-$(uname -m).tar.gz";     wget "https://dl.google.com/dl/cloudsdk/c  0.6s
------
 > [binary_tools_context 90/95] RUN set -eux;     GCLOUD_TAR_FILE="google-cloud-sdk-345.0.0-linux-$(uname -m).tar.gz";     wget "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/${GCLOUD_TAR_FILE}";     tar -xzvf ."/${GCLOUD_TAR_FILE}" -C "/out/usr/local" && rm "${GCLOUD_TAR_FILE}";     /out/usr/local/google-cloud-sdk/bin/gcloud components install beta --quiet;     rm -rf /usr/local/google-cloud-sdk/.install/.backup:
#129 0.362 + uname -m
#129 0.369 + GCLOUD_TAR_FILE=google-cloud-sdk-345.0.0-linux-aarch64.tar.gz
#129 0.369 + wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-345.0.0-linux-aarch64.tar.gz
#129 0.419 --2021-07-12 20:06:17--  https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-345.0.0-linux-aarch64.tar.gz
#129 0.427 Resolving dl.google.com (dl.google.com)... 172.217.164.142, 2607:f8b0:4004:814::200e
#129 0.456 Connecting to dl.google.com (dl.google.com)|172.217.164.142|:443... connected.
#129 0.523 HTTP request sent, awaiting response... 404 Not Found
#129 0.547 2021-07-12 20:06:18 ERROR 404: Not Found.
#129 0.547
------
Dockerfile:260
--------------------
 259 |     # Install gcloud beta component
 260 | >>> RUN set -eux; \
 261 | >>>     GCLOUD_TAR_FILE="google-cloud-sdk-${GCLOUD_VERSION}-linux-$(uname -m).tar.gz"; \
 262 | >>>     wget "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/${GCLOUD_TAR_FILE}"; \
 263 | >>>     tar -xzvf ."/${GCLOUD_TAR_FILE}" -C "${OUTDIR}/usr/local" && rm "${GCLOUD_TAR_FILE}"; \
 264 | >>>     ${OUTDIR}/usr/local/google-cloud-sdk/bin/gcloud components install beta --quiet; \
 265 | >>>     rm -rf /usr/local/google-cloud-sdk/.install/.backup
 266 |
--------------------
error: failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c set -eux;     GCLOUD_TAR_FILE="google-cloud-sdk-${GCLOUD_VERSION}-linux-$(uname -m).tar.gz";     wget "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/${GCLOUD_TAR_FILE}";     tar -xzvf ."/${GCLOUD_TAR_FILE}" -C "${OUTDIR}/usr/local" && rm "${GCLOUD_TAR_FILE}";     ${OUTDIR}/usr/local/google-cloud-sdk/bin/gcloud components install beta --quiet;     rm -rf /usr/local/google-cloud-sdk/.install/.backup]: exit code: 8
```

Comment (howardjohn) [reply docker/build-tools/Dockerfile:264]: Need to change aarch64 -> `arm`

Comment (AWSjswinney) [reply docker/build-tools/Dockerfile]: Got it. I definitely should have checked that myself. I tested it with that change and it looks like it works.

**Review by howardjohn — APPROVED** [review]
/ok-to-test

Thanks!

**Pull request merged**